src/number_theory/cyclotomic/primitive_roots.lean

```diff
@@ -203,6 +205,7 @@ include hζ
 lemma norm_eq_one [is_cyclotomic_extension {n} K L] (hn : n ≠ 2)
   (hirr : irreducible (cyclotomic n K)) : norm K ζ = 1 :=
 begin
+  haveI : is_domain L := ⟨by simp, 1, 0, one_ne_zero⟩,
```

Comment:
Why do you need this, don't you already have this as a variable?

Reply:
oopsies, yes, I was originally messing around with just requiring `no_zero_divisors` (which does work) but it's fiddly and not worth it (and no one uses that anyways). I must've forgotten to remove it - thanks!